ci/kind/test-e2e-kind.sh

```diff
@@ -118,8 +118,8 @@ COMMON_IMAGES_LIST=("registry.k8s.io/e2e-test-images/agnhost:2.29" \
                     "projects.registry.vmware.com/antrea/busybox"  \
                     "projects.registry.vmware.com/antrea/nginx:1.21.6-alpine" \
                     "projects.registry.vmware.com/antrea/perftool" \
-                    "antrea/antrea-ubuntu:latest" \
-                    "antrea/flow-aggregator:latest" \
+                    "projects.registry.vmware.com/antrea/antrea-ubuntu:latest" \
+                    "projects.registry.vmware.com/antrea/flow-aggregator:latest" \
```

Comment:
Any reason to change this back to the harbor ones? I remember we use the docker hub as source for some cache problem in harbor: https://github.com/antrea-io/theia/pull/38

Reply:
I didn't know we had a cache problem before. I was thinking to change that because recently we have an issue when pulling images not from harbor on Jenkins. https://github.com/antrea-io/theia/pull/472

Since we don't have this issue on github, I've reverted it back.